packages/fxa-content-server/app/scripts/views/support.js

```diff
@@ -79,14 +79,7 @@ const SupportView = BaseView.extend({
       if (subscriptions.length) {
         this.model.set('subscriptions', subscriptions);
 
-        const getPlansPromise = account
-          .fetchSubscriptionPlans()
-          .then(plans => {
-            this.model.set('plans', plans);
-          })
-          .catch(err => console.log(err));
-
-        return Promise.all([account.fetchProfile(), getPlansPromise]);
+        return Promise.resolve(account.fetchProfile());
```

Comment:
Could probably just `return account.fetchProfile` directly here, since it returns a promise.

Reply:
Can do